corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -564,13 +564,14 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             self.performSubmit();
```

Comment:
what about this usage of performSubmit() with the param change?

Reply:
This believe this should be fine as `initiatedBy` will be `undefined` in this call, which will be handled [here](https://github.com/dimagi/commcare-hq/pull/33678/files/84a85c89f8e8f82951a7f79b5012e75d9bf770fc#diff-8b8c7bc0774737538270c8660735bf9f33873d5295f70efc4775d86487f01d74R278)